docs/user-interface/graphics/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Graphics"
+description: ".NET MAUI includes cross-platform 2D graphics functionality that targets iOS, Android, Windows, macOS, Tizen, and Linux."
+ms.date: 07/23/2021
+---
+
+# Graphics
+
+<!-- Sample link goes here -->
+
+.NET Multi-platform App UI (.NET MAUI) includes cross-platform 2D graphics functionality that targets iOS, Android, Windows, macOS, Tizen, and Linux. With this functionality, you can use a common API to target multiple graphics abstractions, enabling you to share your drawing code between platforms, or mix and match graphics implementations within a single app.
+
+The graphics functionality is contained in the `Microsoft.Maui.Graphics` namespace, and provides a drawing canvas that supports drawing and painting shapes and images, compositing operations, and graphical object transforms.
+
+There are many similarities between the functionality provided by `Microsoft.Maui.Graphics`, and the functionality provided by .NET MAUI shapes and brushes. However, each is aimed at different scenarios:
+
+- `Microsoft.Maui.Graphics` functionality must be consumed on a drawing canvas, enables performant graphics to be drawn, and provides a convenient approach for writing graphics-based controls. For example, a control that replicates the GitHub contribution profile can be more easily implemented using `Microsoft.Maui.Graphics` than by using .NET MAUI shapes.
+- .NET MAUI shapes can be consumed directly on a page, and brushes can be consumed by all controls. This functionality is provided to help you produce an attractive UI.
+
+## Platform abstractions
+
+The following table lists the graphics abstractions that are supported on each platform:
+
+| Platform | Graphics abstractions |
+| -- | -- |
+| .NET MAUI | Platform support as shown per platform below. |
```

Comment:
I think that we will have a Core library maintained with support for the platforms officially supported by .NET MAUI and another Community library with the rest of the platforms. @Redth could we confirm this? If so, just mention in documentation the officially supported platforms?